app/javascript/components/items/Table.js

```diff
@@ -161,8 +187,8 @@ const Table = (props) => {
           </tr>
         </thead>
         <tbody>
-          {sortedItems.map((item, index) => (
-            <ItemRow key={item.id} item={item} index={index} csrf={csrf} />
+          {sortedItems.map((item, index) => ( 
+            <ItemRow key={item.id} item={item} index={index} csrf={csrf} toggleBought={toggleBought} removeItem={removeItem} />
```

Comment:
Now that QuantityController is in play, we don't need to pass along the index.
```suggestion
            <ItemRow key={item.id} item={item} csrf={csrf} toggleBought={toggleBought} removeItem={removeItem} />
```